roles/openshift_docker_registry/vars/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+
```

Comment:
So I generally agree that dead code or dead files should be removed, however, in this case, removing this file will make the whole vars directory disappear from git since git doesn't allow empty directories.

I'd prefer to leave this here, just so it's easier to add variables to the roles (people don't have to remember "is the dir vars, var, variables, data, etc").